src/coinbase/transaction.ts

```diff
@@ -79,6 +79,15 @@ export class Transaction {
     return this.model.from_address_id;
   }
 
+  /**
+   * Returns the To Address ID for the Transaction.
+   *
+   * @returns The To Address ID
+   */
+  toAddressId(): string {
+    return this.model.to_address_id || "";
```

Comment:
Consider doing something [similar here](https://github.com/coinbase/coinbase-sdk-nodejs/blob/master/src/coinbase/staking_operation.ts#L47-L54) like how the StakingOperation handles an undefined response.

Reply:
Good call out. Done.